src/main/java/seedu/address/model/Transcript.java

```diff
@@ -130,9 +130,13 @@ public void removeModule(Predicate<Module> predicate) {
     //@@author jeremiah-ang
     @Override
     public double getCurrentCap() {
-        return currentCap;
+        return round(currentCap, 1);
     }
 
+    private static double round (double value, int precision) {
```

Comment:
I think the styling could be done on the UI side instead of the Transcript.

Reply:
Yeah I can get it done. I'll create the issue.